cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/reader/StreamScanner.java

```diff
@@ -49,30 +49,36 @@
  * <p>
  * Upon return from the next() call the current values of the scanner can be obtained by calling
  * the methods in Rid, getPartitionKey(), getColumnName(), getValue().
+ *
  * @param <Type> type of object returned by rid() method.
  */
 @SuppressWarnings("unused")
 public interface StreamScanner<Type> extends Closeable
```

Comment:
Can you rename the type parameter from `Type` to `T` in this refactoring? I am strongly -1 on having the same name pattern as the class names. It is confusing when reading the code, mistaking the type parameter for the actual class types.